tf_encrypted/keras/models/sequential_test.py

```diff
@@ -53,14 +53,14 @@ def test_from_config(self):
                                                          input_shape)
 
     with tfe.protocol.SecureNN():
-      tfe_model = tfe.keras.models.Sequential([])
-      tfe_model = tfe_model.from_config(k_config)
       x = tfe.define_private_variable(input_data)
 
-    with tfe.Session() as sess:
-      sess.run(tf.global_variables_initializer())
+      tfe_model = tfe.keras.models.Sequential([])
+      tfe_model = tfe_model.from_config(k_config)
```

Comment:
this is a bit of a strange pattern (instantiate a model just to throw it away); is there an equivalent `from_config` in TF?

Reply:
I think this is just improper usage of the existing method, since it's a classmethod and doesn't require the instantiation here

Reply:
ah correct, the following would work: 
```
tfe_model = Sequential.from_config(k_config)
tfe_model.set_weights(k_weights)
```